test/javascripts/chat-fixtures.js

```diff
@@ -196,7 +195,7 @@ export const chatChannels = {
 function addSettingsAttrs(channel) {
   channel.current_user_membership = channel.current_user_membership || {};
   channel.current_user_membership.unread_count = 0;
-  channel.current_user_membership.unrad_mentions = 0;
+  channel.current_user_membership.unread_mentions = 0;
```

Comment:
Does this typo not affect any features?

Reply:
Most of unread stuff is very badly tested, so iI wouldn't be surprised, this is just setting some general stuff and probably most of it is not used. We can keep it though as it's a valid expectation to have this here, will remove it once we move to fabricators.

Reply:
Hm, actually the whole thing seems to be unused 👀